src/dotnet/APIView/APIViewWeb/appsettings.json

```diff
@@ -1,7 +1,15 @@
 {
   "Logging": {
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Information",
```

Comment:
Do we want default at information level or warning? It's probably a good idea to set it to warning in prod.